config/co2eq_parameters.js

```diff
@@ -49,7 +49,7 @@ exports.fossilFuelAccessor = (zoneKey, k, v) => {
   return (k == 'coal' ||
           k == 'gas' ||
           k == 'oil' ||
-          (k === 'unknown' && (zoneKey !== 'GB-ORK' && zoneKey !== 'UA' && zoneKey !== 'SG' && zoneKey !== 'PR')) ||
+          (k === 'unknown' && (zoneKey !== 'GB-ORK' && zoneKey !== 'UA' && zoneKey !== 'SG' && zoneKey !== 'PR' && zoneKey !== 'FO')) ||
```

Comment:
We are actually in the progress of deprecating this file. So for now we also have to add this to the bottom of co2eq_parameters.json. (https://github.com/tmrowco/electricitymap-contrib/blob/9550a1ff35f2fad6e363c1afb096970f96fec687/config/co2eq_parameters.json#L2949-L3012)

Reply:
Done

Reply:
Also removed a now-unused import